src/sections/offering/price-info.tsx

```diff
@@ -0,0 +1,243 @@
+  fontWeight: 'normal',
+  textAlign: 'center',
+  '@media (max-width: 640px)': {
+    paddingLeft: '0px',
+    paddingRight: '0px',
+  },
+  '@media (max-width: 400px)': {
+    fontSize: '25px',
+  },
+})
+
+export const SubTitle = styled('p')({
+  marginTop: '0px',
+  marginBottom: '0px',
+  marginLeft: 'auto',
+  marginRight: 'auto',
+  fontSize: '16px',
+  textAlign: 'center',
+  maxWidth: '100%',
+  color: colors.DARK_GRAY,
+})
+
+const Price = styled('p')({
+  marginBottom: '0px',
+  marginTop: '30px',
+  fontSize: '20px',
+  fontWeight: 100,
+  textAlign: 'center',
+  color: colors.PURPLE,
+})
+
+const Time = styled('p')({
+  marginBottom: '30px',
+  marginTop: '0px',
+  fontSize: '16px',
+  fontWeight: 100,
+  textAlign: 'center',
+  color: colors.BLACK_PURPLE,
+})
+
+export const Label = styled('p')({
+  marginBottom: '0px',
+  marginTop: '0px',
+  fontSize: '16px',
+  textAlign: 'center',
+  color: colors.DARK_GRAY,
+})
+
+export const ProtectionLabel = styled('p')({
+  marginBottom: '0px',
+  marginTop: '0px',
+  fontSize: '16px',
+  textAlign: 'center',
+  color: colors.BLACK,
+})
+
+export const Row = styled('div')({
+  display: 'flex',
+  alignItems: 'baseline',
+  flexDirection: 'row',
+  justifyContent: 'center',
+  '@media (max-width: 400px)': {
+    flexDirection: 'column',
+    alignItems: 'center',
+  },
+})
+
+export const Col = styled('div')({
+  display: 'flex',
+  alignItems: 'center',
+  flexDirection: 'column',
+  marginBottom: '30px',
+})
+
+const Image = styled('img')({
+  margin: '30px',
+  maxWidth: '70px',
+  '@media (max-width: 400px)': {
+    margin: '0px',
+    maxWidth: '150px',
+  },
+})
+
+export const Title = styled('label')({
+  marginBottom: '10px',
+  fontSize: '16px',
+  textAlign: 'center',
+  color: colors.DARK_GRAY,
+  maxWidth: '150px',
+})
+
+export const GetInsuredButton = styled('div')({
+  display: 'flex',
+  flexDirection: 'row',
+  justifyContent: 'center',
+})
+
+export const LinkTag = styled('a')({
+  backgroundColor: colors.GREEN,
+  fontSize: '18px',
+  color: colors.WHITE,
+  textDecoration: 'none',
+  borderRadius: '50px',
+  padding: '15px 30px',
+})
+
+interface Props {
+  alreadyInsured: boolean
+  header: string
+  subTitle1: string
+  subTitle2: string
+  price: string
+  subscriptionTime: string
+  startDate: string
+  coverage: string
+  getInsured: string
+  iconWorld: any /*TODO: type fix*/
+  iconClock: any /*TODO: type fix*/
+  BackgroundImage: any /*TODO: type fix*/
+  alreadyInsuredLabel: string
+  todayLabel: string
+  protection: string
+}
+
+interface MyComponentState {
+  buttonIsHidden: boolean
+}
+
+export class PriceInfo extends React.Component<Props, MyComponentState> {
+  componentDidMount() {
+    window.addEventListener('scroll', this.hideButton)
+  }
+
+  componentWillUnmount() {
+    window.removeEventListener('scroll', this.hideButton)
+  }
+
+  constructor(props: any) {
+    super(props)
+    this.hideButton = this.hideButton.bind(this)
+    this.state = {
```

Comment:
we could just set `state: State =  { ... }` outside of the constructor, more declarative and neat imo